Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -425,11 +445,14 @@ void ABenchmarkGymGameModeBase::ParsePassedValues()
 
 		FParse::Value(*CommandLine, *MaxRoundTripCommandLineKey, MaxClientRoundTripSeconds);
 		FParse::Value(*CommandLine, *MaxUpdateTimeDeltaCommandLineKey, MaxClientUpdateTimeDeltaSeconds);
+		float fValue = 0.0f;
```

Comment:
fValue, doesn't follow Unreal naming convention... Value of What? pseudo hungarian notation is unnecessary

Reply:
Why do we need this temporary at all, just pass MinActorMigration

Reply:
no, MinActorMigration is double, but `FParse::Value` doesn't support double. So we need to use the temporary variable to convert it.